src/utils.c

```diff
@@ -0,0 +1,35 @@
+
+#include <stdarg.h>
+#define INCL_DOSSEMAPHORES
+#define INCL_DOSMODULEMGR
+#define INCL_DOSMISC
+#define INCL_DOSNLS
+#define INCL_DOSERRORS
+#include <os2emx.h>
+
+
+/* Return TRUE iff the region of memory starting at START is
+   readable. */

```

Comment:
Add proper attribution.
```suggestion
/* Return TRUE iff the region of memory starting at START is
   readable.

   Borrowed from kLIBC src/emx/src/os2/utils.c
   Copyright (c) 1994-1998 by Eberhard Mattes */
```